### PR TITLE
Add a List Tables to HubDB (Another pull request will be made to the HubSpot CLI as well)

### PR DESCRIPTION
**Pull request by 05bmckay**
## Description and Context

Just a quick little function that pulls the published tables from the HubDB.

## TODO

I'll make a pull request to add a `hs hubdb list` command in the CLI as well.

## Who to Notify

@brandenrodgers @joe-yeager @ajlaporte

### CODE REVIEW

File: api/hubdb.ts [hunk 1]
```diff
@@ -11,6 +11,14 @@ import {
 
 const HUBDB_API_PATH = 'cms/v3/hubdb';
 
+export function listTables(
+  accountId: number
+): AxiosPromise<Table> {
```
Comment (brandenrodgers) [inline api/hubdb.ts:16]: The response type here is actually going to look a little different. There's an overview of it [here](https://developers.hubspot.com/beta-docs/reference/api/cms/hubdb#get-%2Fcms%2Fv3%2Fhubdb%2Ftables), but it'll look similar to the existing `FetchRowsResponse` type.

File: api/hubdb.ts [hunk 2]
```diff
@@ -11,6 +11,14 @@ import {
 
 const HUBDB_API_PATH = 'cms/v3/hubdb';
 
+export function listTables(
```
Comment (brandenrodgers) [inline api/hubdb.ts:14]: This is a small nit, but would you mind updating this to be `fetchTables` just to align better with our existing naming ?

Comment (05bmckay) [reply api/hubdb.ts:14]: Good call!

File: api/hubdb.ts (hunk 1 above)
Comment (05bmckay) [reply api/hubdb.ts]: Sorry you're definitely right, forgot to add that type. Thanks!

**Comment by 05bmckay** [comment]
@brandenrodgers made those changes, really appreciate you taking the time for the review.

**Review by brandenrodgers — APPROVED** [review]
🎉 thanks Carter! LGTM

**Pull request merged**